rust_book_ex_8_3/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+pub mod vector_ex;
+pub mod string_ex;
+pub mod hashmap_ex;
+
+#[cfg(test)]
```

Comment:
Btw: you place your tests in `tests/` in root folder, `cargo test` will figure out these are tests automatically. You will then refer to your crate as an external crate (tests will be compiled as a separate crate).